src/ParseServer.js

```diff
@@ -410,6 +415,28 @@ class ParseServer {
   static createLiveQueryServer(httpServer, config) {
     return new ParseLiveQueryServer(httpServer, config);
   }
+
+  static verifyServerUrl(serverUrl, callback) {
+    // perform a health check on the publicServerURL value, with 2.5 second delay
+    setTimeout(function() {
+      if(serverUrl) {
+        const request = require('request');
+        request(serverUrl.replace(/\/$/, "") + "/health", function (error, response, body) {
+          if (error || response.statusCode !== 200 || body !== "OK") {
+            // eslint-disable-next-line
+            console.warn("\nWARNING, Unable to connect to publicServerURL '" + serverUrl + "'. Cloud code and push notifications may be unavailable!\n");
```

Comment:
use template
```
`${a-var} blah`
```

break onto two lines.  
remove the // eslint-disable

Reply:
Totally valid nits. I've made the changes locally but I'm still fiddling with something else before I push changes up.

Reply:
@acinader as for the disable on lint console statements don't look to be allowed. Ending up with `Unexpected console statement  no-console`. If you take a look you'll see that others have run into the same issue and ignored their console calls as well. 

If there's some alternative approach I'm not aware of let me know, but that seems about right.

Reply:
Following up on that something like `/* eslint-disable no-console */` may be more appropriate than just a flat out disable.

Reply:
ah, yeah.  the logger isn't available here.  so yes, adding the error your targeting (no-console) is the right call see around line 225 where console is used.